app/api/v2/handlers/operation_api.py

```diff
@@ -68,7 +68,7 @@ async def get_operation_by_id(self, request: web.Request):
                           summary='Create a new CALDERA operation record',
                           description='Create a new CALDERA operation using the format provided in the '
                                       '`OperationSchema`. Required nested schema fields are as follows: '
```

Comment:
Sorry for such a nitpicky comment, but can you remove "nested" from this line? just want to avoid potential confusion as `name` is not nested within one of the objects in the operation schema